### ISSUE TITLE
Create a page in our documentation about accessibility of docs.ansible.com, how we aim to enforce it and who/where to reach for questions/concerns

### ISSUE THREAD

**Issue by Spredzy** [ansible-community/community-team#44]
Accessibility is important. Once we improve how accessible our documentation is, we should be proud of it and have a dedicated page in our documentation to talk about it. Why we care, what we do to ensure continuous accessibility, etc...

**Expected Results**

A page in docs.ansible.com that explains our position and practices around accessibility
